### ISSUE TITLE
Data Migration: Search date facet cannot find any migrated studies with dates.

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#36]
---

Author Name: **Kevin Condon** (@kcondon)
Original Redmine Issue: 3443, https://redmine.hmdc.harvard.edu/issues/3443
Original Date: 2014-01-27

---

Search facet for dates does not find any migrated studies.

The facet for dates does find created studies. It indexes production date. There seems to be two issues: 1. Something about migration that prevents dates from appearing in facet, 2. date facet possibly not checking the correct date field.

There is currently an issue with the date field on the dataset page that came up in Merce's pilot testing and which will be added in as separate ticket:

The date field is vague and should say what type of date it is: Citation, Distribution, etc. Second, it should be automatically generated and not appear at all. Third, it should be the date it is shared with others. Last, since it is possible it has been shared outside Dataverse, there should be the ability to edit/override it.

So, bottom line is we need to agree which date field should be displayed by date facet and that is affected by how the citation date is derived if citation date is what is displayed. 

---

Related issue(s): #95
Redmine related issue(s): [3504](https://redmine.hmdc.harvard.edu/issues/3504)

---
